.github/ISSUE_TEMPLATE/---bug-report.md

```diff
@@ -0,0 +1,20 @@
+---
+name: "\U0001F41B Bug report"
+about: "If something isn't working as expected \U0001F914."
```

Comment:
The table specifies how the issue is displayed on the new issue page:
<img width="496" alt="screenshot 2018-08-15 at 14 14 31" src="https://user-images.githubusercontent.com/8514706/44147765-b741a56e-a095-11e8-83bc-ec8dbdf2483e.png">